tests/unit/model/a_p_i_query_test.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from pyatlan.model.assets import APIQuery
+from tests.unit.model.constants import (
+    API_CONNECTION_QUALIFIED_NAME,
+    API_CONNECTOR_TYPE,
+    API_QUERY_QUALIFIED_NAME,
+    API_QUERY_NAME,
+    API_QUERY_REFERENCE_OBJECT_QN,
+)
+
+
+@pytest.mark.parametrize(
+    "name, connection_qualified_name, message",
+    [
+        (None, "connection/name", "name is required"),
+        (API_QUERY_NAME, None, "connection_qualified_name is required"),
+    ],
+)
+def test_create_with_missing_parameters_raise_value_error(
+    name: str, connection_qualified_name: str, message: str
+):
+    with pytest.raises(ValueError, match=message):
+        APIQuery.create(name=name, connection_qualified_name=connection_qualified_name)
+
```

Comment:
We should use `creator()` instead (ref: https://github.com/atlanhq/atlan-python/pull/403/files#r1800760792)